teamscale_client/constants.py

```diff
@@ -123,19 +145,51 @@ class ReportFormats:
     SPCOP = "SPCOP"
 
     CS_COMPILER_WARNING = "CS_COMPILER_WARNING"
-   
+
     PYLINT = "PYLINT"
 
     FINDBUGS = "FINDBUGS"
 
-class UnitTestReportFormats:
+    CPPCHECK = "CPPCHECK"
+
+    MODEL_ADVISOR = "MODEL_ADVISOR"
+
+    SAP_CODE_INSPECTOR = "SAP_CODE_INSPECTOR"
+
+    PARASOFT_CPP_TEST = "PARASOFT_CPP_TEST"
+
+    MYPY = "MYPY"
+
+    JQASSISTANT = "JQASSISTANT"
+
+    GENERIC_FINDINGS = "GENERIC_FINDINGS"
+
+
+class UnitTestReportFormats(str, Enum):
     """Reports for unit test results that Teamscale understands."""
 
     JUNIT = "JUNIT"
 
     XUNIT = "XUNIT"
 
-class ConnectorType:
+    MS_TEST = "MS_TEST"
+
+    TESTWISE_COVERAGE = "TESTWISE_COVERAGE"
```

Comment:
As this is a combined format users might also expect to find it under CoverageFormats? Can we just list it in both places?

Reply:
Already in, line 121 :)

But I double checked for the rest.